docs/disconnectedCallback.md

```diff
@@ -0,0 +1,28 @@
+@typedef {function} can-component/disconnectedCallback disconnectedCallback
+@parent can-component.events
+
+@description An event called with the component’s element during teardown after bindings are removed.
+
+@signature `connectedCallback: () => function disconnectedCallback (element) { ... }`
```

Comment:
This signature seems weird to me; should it be `connectedCallback: function disconnectedCallback(element) { ... }`?

Reply:
It's the function that's returned from the connectedCallback function. It's not the best signature in the world but it shows what's needed.